dev/FHIR3-0-2-MM202001-Test/BgZ-extended-3-0/_reference/resources/medmij-bgz-patient-ts-01.xml

```diff
@@ -40,11 +40,6 @@
             <valueCode value="CL"/>
          </extension>
       </given>
-      <given value="J.">
-         <extension url="http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier">
-            <valueCode value="IN"/>
-         </extension>
-      </given>
```

Comment:
Weten we zeker dat deze weg moet? Daarvoor zouden dan denk ik ook de addenda moeten worden aangepast. Misschien voor de kwalificatiematerialen niet weghalen?